src/routes/login.ts

```diff
@@ -135,20 +135,22 @@ export const createLoginRoute: RouteCreator =
           )
         }
 
+        let initRecoveryUrl = ""
         const initRegistrationUrl = getUrlForFlow(
           kratosBrowserUrl,
           "registration",
           initRegistrationQuery,
         )
-
-        const initRecoveryUrl = getUrlForFlow(
-          kratosBrowserUrl,
-          "recovery",
-          new URLSearchParams({
-            return_to:
-              (return_to && return_to.toString()) || flow.return_to || "",
-          }),
-        )
+        if (flow.refresh) {
+          initRecoveryUrl = getUrlForFlow(
+            kratosBrowserUrl,
+            "recovery",
+            new URLSearchParams({
+              return_to:
+                (return_to && return_to.toString()) || flow.return_to || "",
+            }),
+          )
+        }
```

Comment:
what happens if we don't refresh the flow at all and we are logging in for the first time? Then the recovery link will be gone right?

Reply:
A better way might be to retrieve the session and then check if the user is already logged in. Based on that + refresh true we can hide the recovery link. 

We will also need to omit the `forgotPasswordURL` property from `additionalProps` in the `UserAuthCard` function since we don't want to show the link at all.